cost/kubecost/cluster/kubecost_cluster_sizing_recommendations_cust.pt

```diff
@@ -0,0 +1,201 @@
+name "Kubecost Cluster Sizing Recommendations cust"
+rs_pt_ver 20180301
+type "policy"
+short_description "Pulls the Cluster Sizing Recommendations from Kubecost and raises an incident. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/kubecost/cluster/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
The policy retrieves Kubecost recommendations for rightsizing of Kubernetes clusters and ...

Reply:
Updated short description